### PR TITLE
Deep Learning, philosophical intro

### PR DESCRIPTION
**Pull request by MarcoArmenta**

### CODE REVIEW

File: article/_posts/2020-06-19-DL_philosophy.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+layout: review
+title: "Deep Learning: A philosophical introduction"
+tags: deep-learning
+author: "Marco Armenta"
+cite:
+    authors: "Cameron Buckner"
+    title:   "Deep Learning: A philosophical introduction"
+    venue:   "Philosophy Compass"
```
Comment (jhlegarreta) [inline article/_posts/2020-06-19-DL_philosophy.md:9]: Is this an open journal? If not, we may not be allowed to reproduce the images.

File: article/_posts/2020-06-19-DL_philosophy.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+![](/article/images/Philosophy_DL/feature_space.jpg)
+
+![](/article/images/Philosophy_DL/folding_ex.jpg)
+
+# Open Problems
+
+Most commentators agree that current deep learning methods fall short of implementing general intelligence, and it remains an open question as to whether some modification of current deep learning methods will be able to do so.
+
+* **CNNs and nativism/empiricism**: Nativism is a concept in psychology and philosophy which asserts certain concepts are "native" or in the brain at birth (given a priori). Empiricism is a theory that states that knowledge comes only or primarily from sensory experience. Philosophers think of cognition as ranging over four variables: algorithms, representational formats, innate knowledge and experience. A truly empiricist system should begin with nothing for representational format and innate knowledge, and only a very minimal amount of algorithms, deriving everything else from experience. Therefore, AlphaGo, AlphaStar etc. provide philosophical insight into empiricism.
```
Comment (jhlegarreta) [inline article/_posts/2020-06-19-DL_philosophy.md:44]: _Nativism is a concept in psychology and philosophy which asserts certain concepts are "native" or in the brain at birth (given a priori)_.  : is the `or` necessary?

**Comment by AntoineTheb** [comment]
Will merge once comments are addressed

File: article/_posts/2020-06-19-DL_philosophy.md (hunk 1 above)
Comment (jhlegarreta) [reply article/_posts/2020-06-19-DL_philosophy.md:9]: Addressed in bf1121e.

File: article/_posts/2020-06-19-DL_philosophy.md (hunk 2 above)
Comment (jhlegarreta) [reply article/_posts/2020-06-19-DL_philosophy.md]: Addressed in bf1121e.

**Pull request merged**